### ISSUE TITLE
Em dash isn't treated as word divider by the dictionary

### ISSUE THREAD

**Issue by a-kohout** [koreader/koreader#2634]
* KOReader version: v2015.11-896-g4316284
* Device: Kobo Glo HD

#### Issue

Em dash isn't treated as word divider by the dictionary.
Long-pressing the word "surreptitiously" in phrase "to work surreptitiously—and very"
looks up "surreptitiouslyand"

relevant crash.log entry:
```
Nothing similar to surreptitiouslyand, sorry :(
03/12/17-21:53:00 WARN  JSON data cannot be decoded
```
#### Steps to reproduce
[demo.epub.zip](https://github.com/koreader/koreader/files/836957/demo.epub.zip)

**Comment by retrue** [comment]
Duplicated
This issue was already reported #2413

**Comment by Frenzie** [comment]
Duplicate and fixed in #2708.
